### DISCUSSION TITLE
whisper "forgets" to translate during transcription

### DISCUSSION THREAD

**Discussion by jstaerk** [openai/whisper#541] [Q&A]
When I invoke 
`whisper --model large --task translate` on https://www.youtube.com/watch?v=Iqx8cdsNnXE
Only the first four words are translated(?), the output looks like

```
Then I call on Tagesordnungspunkt 81,
Antrag Aktuelle Stunde der Fraktion der SPD,
nicht nur Bürgerbusse feiern, sondern Infrastruktur in ländlichen Räumen sichern.
```
This is apparently not related to the recognized language, it's german, but even when I explicitly set this using
`--language German` it turns off the translation after the first words

**Comment by BrightXiaoHan** [comment]
Hi! 
I meet the same problem, have you found any answer?

**Reply by jstaerk** [reply]
not yet :-(
